Program.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
```

Comment:
Missing a demo.

Reply:
i accidentally added a second main. demo was in the original.